### PR TITLE
migration - Move from deta.space to cloudflare kv

### PR DESCRIPTION
**Pull request by ashishjullia**
Changes:
- The OTP logic is directly handled by [otp.js](https://github.com/ashishjullia/portunus-worker/blob/move-deta-cf-kv/modules/otp.js)
- Because deta.space is shutting down (ref: https://deta.space/sunset), it is required to move away from it as the services will stop on October 17, 2024. So, to solve this, the code has been refactored to completely rely on Cloudflare worker KV.
- Dev workflow is also updated but requires owner's update to it for specifying the new KV namespace bindings and make a deployment and then proceed with the testing.

### CODE REVIEW

File: handlers.js [hunk 1]
```diff
@@ -76,43 +75,52 @@ module.exports.root = ({ headers, cf }) =>
   respondJSON({
     payload: {
       cli: 'pip install -U print-env --pre',
-      'Web UI': 'https://portunus.netlify.app',
+      'Web UI': 'https://portunus.locus.engineering',
```
Comment (woozyking) [inline handlers.js:78]: Can we make this configurable through env var?

Comment (ashishjullia) [reply handlers.js]: done

Comment (ashishjullia) [reply handlers.js]: It will be better to put the value of this variable in `' '` while putting in gh s.ecrets

**Review by woozyking — APPROVED** [review]

File: handlers.js (hunk 1 above)
Comment (woozyking) [reply handlers.js]: Done!

**Comment by woozyking** [comment]
Will fix the error in the future

**Pull request merged**